frontend/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_GEOAPIFY_KEY="a6e300938f344f0f97d864202683b815"
```

Comment:
It's nice that you use a .env file to store your api keys! You should put the env in your .gitignore file though since you don't want to share your api keys with the rest of the world (at least most of the time). Doesn't matter that much since it ain't a db key or something but just fyi.